src/config/api.js

```diff
@@ -0,0 +1,9 @@
+import { create } from 'apisauce';
+
+const api = create({
+  // baseURL: process.env.API_BASE_URL,
```

Comment:
We have to do work this.
Delete the  comment and try to use de env

Reply:
Done.
Variables in .env must start with "REACT_APP_ *" to work